code/__defines/math_physics.dm

```diff
@@ -19,7 +19,6 @@
 #define T20C 293.15 //   20.0 degrees celcius
 #define TCMB 2.7    // -270.3 degrees celcius
 
-#define CLAMP01(x) max(0, min(1, x))
```

Comment:
Maybe it would be better to keep this one and delete the one in `macros.dm`.

Reply:
Benchmarking seems to show this one is faster too.

Reply:
One would expect inlining to be faster. But this is BYOND, so who knooows.